src/waveform/renderers/waveformmarkset.cpp

```diff
@@ -35,9 +35,9 @@ void WaveformMarkSet::setup(const QString& group, const QDomNode& node,
             WaveformMarkPointer pMark(new WaveformMark(group, child, context, signalColors));
 
             bool uniqueMark = true;
-            if (pMark->m_pPointCos) {
+            if (!pMark->is_m_pPointCos_Null()) {
```

Comment:
When does it happen that m_pPointCos  is null? Can this happen? 
I would be nice if we can get rid of is_m_pPointCos_Null() and handle this case in getConfigKey() 
Or catch the case somehow earlier that it cannot happen.